### PR TITLE
add a space + buster buster

### PR DESCRIPTION
**Pull request by jodumont**
it's works now on Buster

### CODE REVIEW

**Comment by ydakuka** [comment]
@adamierymenko, can you merge it?

**Comment by jodumont** [comment]
@ydakuka I did a remake today
https://github.com/zerotier/install.zerotier.com/pull/7
which I also make it works with kali

**Comment by adamierymenko** [comment]
Buster works, so closing... this is old. Should check these other GitHub projects more often. 😦

**Comment by KyleChou** [comment]
sorry, i didn't understand
where should i add space???

File: htdocs/install.sh [hunk 1]
```diff
@@ -172,9 +172,9 @@ if [ -f /etc/debian_version ]; then
 	elif [ "$dvers" = "9" -o "$dvers" = "stretch" ]; then
 		echo '*** Found Debian "stretch" (or similar), creating /etc/apt/sources.list.d/zerotier.list'
 		echo "deb ${ZT_BASE_URL_HTTP}debian/stretch stretch main" >/tmp/zt-sources-list
-	elif [ "$dvers" = "10" -o "$dvers" = "11" -o "$dvers" = "sid" -o "$dvers" = "buster"]; then
+	elif [ "$dvers" = "10" -o "$dvers" = "11" -o "$dvers" = "sid" -o "$dvers" = "buster" ]; then
```
Comment (JOduMonT) [inline htdocs/install.sh]: the space goes here at the line 175

**Pull request closed without merging**